scripts/process_data.py

```diff
@@ -276,6 +281,13 @@ def copy_images_list(image_paths: List[Path], image_dir: Path, verbose) -> List[
         shutil.copy(image_path, copied_image_path)
         copied_image_paths.append(copied_image_path)
 
+    if crop_border_pixels is not None:
+        file_type = image_paths[0].suffix
+        filename = f"frame_%05d{file_type}"
+        crop = f"crop=iw-{crop_border_pixels}:ih-{crop_border_pixels}"
+        ffmpeg_cmd = f"ffmpeg -y -i {image_dir / filename} -q:v 2 -vf {crop} {image_dir / filename}"
```

Comment:
I'm not familiar with ffmpeg syntax so just want to be sure -- will this do a center crop to remove `crop_border_pixels/2` from the top and bottom of the image? If not we'll want to modify this to do that.

Reply:
Yes it should. The default origin is the center. This can be confirmed by looking at the post cropped images.